cmd/kubeadm/app/util/runtime/runtime.go

```diff
@@ -108,9 +108,7 @@ func (runtime *CRIRuntime) ListKubeContainers() ([]string, error) {
 	}
 	pods := []string{}
 	for _, pod := range strings.Fields(string(out)) {
-		if strings.HasPrefix(pod, "k8s_") {
-			pods = append(pods, pod)
-		}
+		pods = append(pods, pod)
```

Comment:
Why removing the check?

Reply:
@dixudx `crictl pods -q` will return just pod IDs. those are `[0-9a-f]+`. There are no `k8s_` prefixes there. This prefix is only relevant for listing containers in case of docker as a runtime.

Reply:
@kad Thanks. Got it. We are now using containerd. It's safe to remove this.